src/map/clif.cpp

```diff
@@ -3670,16 +3670,16 @@ void clif_updatestatus( map_session_data& sd, enum _sp type ){
 			break;
 #else
 		case SP_BASEEXP:
-			clif_par_change(sd, type, client_exp(sd.status.base_exp));
+			clif_longpar_change(sd, type, client_exp(sd.status.base_exp));
```

Comment:
Not sure if anyone has more packet data info, but shouldn't this be adjusted to whenever the client changed from `clif_par_change` to `clif_longpar_change` and not thrown into the final else?

Reply:
> Not sure if anyone has more packet data info, but shouldn't this be adjusted to whenever the client changed from `clif_par_change` to `clif_longpar_change` and not thrown into the final else?

ZC_PAR_CHANGE and ZC_LONGPAR_CHANGE have been there since the alpha clients, so they're not date dependent, but depends on the stuff they send.
clif_longpar_change seems correct for exp values